tests/docker/codeception/docker-entrypoint.sh

```diff
@@ -46,7 +46,18 @@ else
 fi
 
 # install WooCommerce (activate & deactivate it to populate DB for backup)
-wp plugin install woocommerce
+if [[ ! -d "/wp-core/wp-content/plugins/woocommerce" ]]; then
```

Comment:
Does this happen every single time when running acceptance tests? That means at least 5 times per build. Shall we do this functionality in the `build` phase and then share it in all acceptance test jobs?

Reply:
I will try it now.